File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.cs
using System;
using System.Collections.Generic;

public class Solution {
    public int[] solution(string my_string) {
        List<int> list = new List<int>();
        
        for(int i = 0; i < my_string.Length; i++)
        {
            if(char.IsDigit(my_string[i]))
            {
                list.Add(int.Parse(my_string[i].ToString()));
            }
        }
        list.Sort();
        return list.ToArray();
    }
}

File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

vector<int> solution(string my_string) {
    vector<int> answer;
    for(char c : my_string)
    {
        if(isdigit(c))
        {
            answer.push_back(c - '0');
        }
        sort(answer.begin(), answer.end());
    }
    return answer;
}